docs/topics/channels/deploying.txt

```diff
@@ -0,0 +1,196 @@
+Deploying
+=========
+
+Deploying applications using Channels requires a few more steps than a normal
```

Comment:
I think we should probably lower-case it; I'll modify the auto-patcher that translates this all over from Channels source code to make the change.